copier/template.py

```diff
@@ -168,12 +168,17 @@ class Task:
         working_directory:
             The directory from inside where to execute the task.
             If `None`, the project directory will be used.
+
+        failure_message:
+            If provided, show the message when a task fails.
+            If not provided, show the exception message from `subprocess`.
```

Comment:
Perhaps use the wording from the documentation for consistency:

```suggestion
            Provides a message to print if the task fails.
            If not provided, the subprocess exception message will be shown.
```